extension/storage/redisstorageextension/factory.go

```diff
@@ -24,6 +25,9 @@ func NewFactory() extension.Factory {
 func createDefaultConfig() component.Config {
 	return &Config{
 		Endpoint: "localhost:6379",
+		TLS: configtls.ClientConfig{
+			Insecure: true,
```

Comment:
typically configtls is setting insecure to false by default. see https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/configtls.go#L84

Reply:
Since the component did not support TLS before, I had set the default to true. It's now defaulted to false.